EpsilonAuxBMS/AuxBMSTests.adoc

```diff
@@ -0,0 +1,102 @@
+= Aux BMS Tests
+
+== HeartBeat Test
+----
+Elapsed 5000ms{
+	Expect 5 Heartbeat CAN messages (0x720)
+}
+----
+
+== Aux Test
+----
+Elapsed 1000ms{
+	Expect 10 Aux CAN messages (0x721)
+}
+----
+
+== CAN Values to Test
+[options="header"]
+|===
+| Input Type | CAN_ADDRESS | DATA_INDEX | TIME_FRAME
+| PrechargeState |  0x721  | 0 |  100ms 
+| AuxVoltage |  0x721 |  0 |  100ms |
+|===
+
+== Precharge Test
+----
+Send (PRECHARGE_CAN_MESSAGE(0x721)[DATA_INDEX])
+Within 100ms {
+    Expect (PRECHARGE_STATUS_CAN_MESSAGE(0x721)[DATA_INDEX])
+}
+----
+
+== AuxVoltage Test
+----
+Every 100ms{
```

Comment:
This isn't really a good for a test because it doesn't really state how long to test it for. The format of the aux test is better.